### PR TITLE
fix: render bands properly not nested bands data

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

```js
const options = {
  plot: {
    bands: [{
      range: [3, 8], // not working
      range: [[3, 8]], // working
    }]
  }
}
```

* render bands properly not nested bands data

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다!

File: apps/chart/src/store/plot.ts [hunk 1]
```diff
@@ -58,29 +58,22 @@ function makePlotLines(categories: string[], isDateType: boolean, plotLines: Plo
...
+      };
     }
 
-    return {
+    return ranges.map((rangeData) => ({
+      range: rangeData,
       color,
-      range,
-    };
+    }));
```
Comment (js87zz) [inline apps/chart/src/store/plot.ts:76]: 위에서 `ranges` 데이터를 만들때부터 color를 같이 포함시켜주면 안되나요~?

**Review by js87zz — APPROVED** [review]
리뷰 완료입니다~!

**Review by jungeun-cho — APPROVED** [review]
확인했습니다~!

File: apps/chart/src/store/plot.ts (hunk 1 above)
Comment (jung-han) [reply apps/chart/src/store/plot.ts:76]: 수정했습니다!

**Pull request merged**